src/index.js

```diff
@@ -205,18 +217,27 @@ const processResults = ({ summaries, errors }) => {
       error,
     };
   } else {
+    const reports = [];
     return {
-      summary: summaries
-        .map(({ path, url, summary }) => {
+      summary: data
+        .map(({ path, url, summary, report }) => {
+          let obj = {};
+          obj = { summary, report };
+
           if (path) {
+            obj.path = path;
+            reports.push(obj);
             return `Summary for directory '${chalk.magenta(path)}': ${summary}`;
           }
           if (url) {
+            obj.url = url;
+            reports.push(obj);
             return `Summary for url '${chalk.magenta(url)}': ${summary}`;
           }
           return `${summary}`;
         })
         .join('\n'),
+      extraData: reports,
```

Comment:
Have you contemplated an object instead of an array? It will give us more flexibility for different schemas.

Reply:
I think an array here makes sense since it is for multiple audits, unless we want an object holding arrays. Currently this is what this will look like:

```
[
    {
        "url": "some_audited_url_A",
        "report": "the report html",
        "summary": "the report summary"
    },
    {
        "url": "some_audited_url_B",
        "report": "the report html",
        "summary": "the report summary"
    },
]
```

What did you have in mind?

Reply:
I was thinking about average, are we planning to compute it on the server side?

Reply:
yes! that is what I was going for